packages/dds/tree/src/test/feature-libraries/flex-tree/editableTreeTypes.spec.ts

```diff
@@ -72,8 +70,7 @@ describe("flexTreeTypes", () => {
 			} else if (tree.is(leaf.string)) {
 				const s: string = tree.value;
 			} else if (tree.is(jsonArray)) {
-				const a: FlexTreeSequenceField<typeof jsonRoot> = tree.content;
-				jsonExample(a);
```

Comment:
Not sure what `jsonExample` did here - no longer needed?

Reply:
this code is an example of this schema aware API's typing. As this schema aware API no longer exists, it has no reason to recursively calls itself to demonstrate is strong typing